go.sum

```diff
@@ -85,6 +85,25 @@ github.com/DATA-DOG/go-sqlmock v1.3.3/go.mod h1:f/Ixk793poVmq4qj/V1dPUg2JEAKC73Q
 github.com/DATA-DOG/go-sqlmock v1.4.1/go.mod h1:f/Ixk793poVmq4qj/V1dPUg2JEAKC73Q5eFN3EC/SaM=
 github.com/DataDog/agent-payload v4.49.0+incompatible h1:CKWb5v7M6o3b5JHbX43qs1bswTri8LH7cmfIKQrEtyk=
 github.com/DataDog/agent-payload v4.49.0+incompatible/go.mod h1:/2RW4IC/2z54jtB6RLgq5UtVI1TsX0joDRjKbkLT+mk=
+github.com/DataDog/agent-payload v4.50.1-0.20210106121336-178e5146827a+incompatible h1:swf51ckdS5clYp/fQ3ZTWKXpf6q1X2hSCgzo5JEElns=
+github.com/DataDog/agent-payload v4.50.1-0.20210106121336-178e5146827a+incompatible/go.mod h1:/2RW4IC/2z54jtB6RLgq5UtVI1TsX0joDRjKbkLT+mk=
+github.com/DataDog/agent-payload v4.50.1-0.20210112172606-7ed069797386+incompatible h1:k0jUApgKX2tTW1KCsGh/oCda5QTANKDOk+ugiyq7yRY=
+github.com/DataDog/agent-payload v4.50.1-0.20210112172606-7ed069797386+incompatible/go.mod h1:/2RW4IC/2z54jtB6RLgq5UtVI1TsX0joDRjKbkLT+mk=
+github.com/DataDog/agent-payload v4.50.1-0.20210113032057-dbdb23b7f873+incompatible h1:kBReCAEm6lWaOdmnDqPymUae2TsD7k0BGZHZJxOKHfE=
+github.com/DataDog/agent-payload v4.50.1-0.20210113032057-dbdb23b7f873+incompatible/go.mod h1:/2RW4IC/2z54jtB6RLgq5UtVI1TsX0joDRjKbkLT+mk=
+github.com/DataDog/agent-payload v4.53.0+incompatible h1:M//H0sBfc8AnLPtziNunUElwq3+vggAKT4AcIFIhr3s=
+github.com/DataDog/agent-payload v4.53.1-0.20210113155314-57fe12249b34+incompatible h1:r8tdmAhzhwTv0QxbwSU78eVSoegPSVyPuIT8Ynitiwc=
+github.com/DataDog/agent-payload v4.53.1-0.20210113155314-57fe12249b34+incompatible/go.mod h1:/2RW4IC/2z54jtB6RLgq5UtVI1TsX0joDRjKbkLT+mk=
+github.com/DataDog/agent-payload v4.53.1-0.20210113164804-44ac42aba77a+incompatible h1:bW0tRyETOim0ycQAUBaANkYLhNTghJM3jOoDtDQjAek=
+github.com/DataDog/agent-payload v4.53.1-0.20210113164804-44ac42aba77a+incompatible/go.mod h1:/2RW4IC/2z54jtB6RLgq5UtVI1TsX0joDRjKbkLT+mk=
+github.com/DataDog/agent-payload v4.53.1-0.20210113165348-51de575020e8+incompatible h1:gXJcHEIpjmrM0WX7oYUjDkZ05z4bqW+vuIvMKInp6fo=
+github.com/DataDog/agent-payload v4.53.1-0.20210113165348-51de575020e8+incompatible/go.mod h1:/2RW4IC/2z54jtB6RLgq5UtVI1TsX0joDRjKbkLT+mk=
+github.com/DataDog/agent-payload v4.53.1-0.20210113193053-81c4274e5061+incompatible h1:Qm+zDmtCKUnGH5AkmgyZv2ankfXVzR4Md9Yp5z5n4W4=
+github.com/DataDog/agent-payload v4.53.1-0.20210113193053-81c4274e5061+incompatible/go.mod h1:/2RW4IC/2z54jtB6RLgq5UtVI1TsX0joDRjKbkLT+mk=
+github.com/DataDog/agent-payload v4.53.1-0.20210114152954-be676bf33059+incompatible h1:RbpQgkQIFyCRi3RMj5PCNmbb/g0u7AW2fDyKltOYWEs=
+github.com/DataDog/agent-payload v4.53.1-0.20210114152954-be676bf33059+incompatible/go.mod h1:/2RW4IC/2z54jtB6RLgq5UtVI1TsX0joDRjKbkLT+mk=
+github.com/DataDog/agent-payload v4.53.1-0.20210114154107-05ad7aa9158d+incompatible h1:K62xDSk0vuDe64XpB1UdGJlT0X4YfrYyyBVYflqWnUM=
+github.com/DataDog/agent-payload v4.53.1-0.20210114154107-05ad7aa9158d+incompatible/go.mod h1:/2RW4IC/2z54jtB6RLgq5UtVI1TsX0joDRjKbkLT+mk=
```

Comment:
A small nit but I think the `.sum` file only needs one version's hashes so I think you can delete the v4.49 and 4.50.1 hashes from this file